Formula/freeimage.rb

```diff
@@ -40,6 +38,19 @@ def install
     system "make", "-f", "Makefile.fip"
     system "make", "-f", "Makefile.fip", "install", "PREFIX=#{prefix}"
   end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <FreeImage.h>
+      int main() {
+         FreeImage_Initialise(0);
+         FreeImage_DeInitialise();
+         exit(0);
+      }
+    EOS
+    system ENV.cc, "test.c", "-L#{lib}", "-lfreeimage", "-o", "test"
```

Comment:
You should make this `system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-lfreeimage", "-o", "test"` so that the test still works when `freeimage` is unlinked.